lib/models/BucketInfo.ts

```diff
@@ -868,4 +909,25 @@ export default class BucketInfo {
         this._tags = tags;
         return this;
     }
+
+    /**
+     * Get the value of bucket capabilities
+     * @param capability? - if provided, will return a specific capacity
+     * @return - capabilities of the bucket, or null
+     */
+    getCapabilities(capability?: string) {
+        if (capability && this._capabilities && this._capabilities[capability]) {
+            return this._capabilities[capability];
+        }
+        return this._capabilities;
```

Comment:
would  it be better to either have 2 functions, to either get a single capability or get the whole list?

i fear typescript may not properly infer typing which depend on the parameter, and using this function may essentially be the same as having `any` type (or a choice of any capability) : where doing `this._capabilites?.<capability>` (or dedicated helper function like `getVeeamSosApi()` has a specific type....

Reply:
Sure, I will create two functions, it will also let me have the right naming

Reply:
I created this commit: https://github.com/scality/Arsenal/pull/2059/commits/f17d52b60281b8f33600a3b60816ffc5d482a918

Keeping this PR open for now, please feel free if you were thinking of something else!